BraveRewardsExample/ViewController.swift

```diff
@@ -70,6 +70,14 @@ let stateStoragePath: String = (NSSearchPathForDirectoriesInDomains(
     true
   ).first! as NSString).appendingPathComponent("brave_ledger")
 
+func rewardsLog(logLevel: LogLevel, line: Int32, file: String, data: String) {
+  if !data.trimmingCharacters(in: .whitespacesAndNewlines).isEmpty {
+    // Should probably just trim the final trailing newline instead, since `print` only appends
+    // a single newline terminator
+    print("[\(logLevel.logPrefix)] \(data.trimmingCharacters(in: .newlines))")
```

Comment:
Do we need to implement it on brave-ios go get control of rewards logs?

Reply:
Yes, though the brave-ios side will use XCLogger instead of `logPrefix`

Reply:
re: os_log vs print, for this case print is fine because this is just an example/host for development, and won't be used in production ever